### ISSUE TITLE
Q&V: can't copy anything from the query results onto the clipboard.

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2938]
vaadin-grid or vaadin-grid-cell needs to have user-select: text.

**Comment by urapadmin** [comment]
### Kiosk 1.6.10.1
- now that should work

**Comment by luizaogs** [comment]
I'm sorry, I'm dumb. Is this not what #2904 was? If not, what am I looking for? 

I do note that I cannot copy the text in the description in the query results below (for instance), but I don't know if this is what I'm supposed to be looking at: 

![Image](https://github.com/user-attachments/assets/8b9d178d-f69d-4f4a-bda7-832b8f335a14)

**Comment by urapadmin** [comment]
I thought the other one was only about View and not about the query results themselves

**Comment by urapadmin** [comment]
> I'm sorry, I'm dumb. Is this not what #2904 was? If not, what am I looking for? 
> 
> I do note that I cannot copy the text in the description in the query results below (for instance), but I don't know if this is what I'm supposed to be looking at: 
> 
> ![Image](https://github.com/user-attachments/assets/8b9d178d-f69d-4f4a-bda7-832b8f335a14)
> 

that's what should work now. But I might have forgotten to include this in the last version

**Comment by luizaogs** [comment]
Would refreshing the cache potentially help? I didn't.

**Comment by luizaogs** [comment]
> I thought the other one was only about View and not about the query results themselves

I confuse the terminology. I guess you're right.

**Comment by luizaogs** [comment]
Clearing the cache did not help.

**Comment by urapadmin** [comment]
tested this myself on BUAP. It kinda works but I am not happy with it. It is not possible to just select a whole column or something. Fickle.

**Comment by urapadmin** [comment]
### Kiosk 1.6.24
- Should work in both Safari and Chrome just fine
- In FireFox you can only select text in a single table cell. If you want to select multiple table cells, you have to click at the beginning of your selection and then - with the shift key pressed - at the end of your selection. That way you can select multiple rows, cells, whatever. It is more or less what happens in Safari and Chrome when you drag-select with the mouse. The shift key selection works in Chrome as well.
- note that we actually have two different query results: The results of the full text search are a different layout than the results of the other queries

**Comment by luizaogs** [comment]
This is working for queries (though I'm having trouble with Firefox - it is fickle and doesn't always work). 

I need #3146 to test the FTS part.

**Comment by urapadmin** [comment]
Ja, FireFox is fickle about selecting in a table. But that is really beyond my influence and FireFox isn't very common among our users.
